### ISSUE TITLE
Training settings

### ISSUE THREAD

**Issue by wuy202** [KyanChen/RSPrompter#8]
Nice work, thanks for sharing. The max epochs and learning rate utilized in this repo are different from the ones stated in your Arxiv paper. For instance, total training epoch is 700/1000 (WHU) in the paper while it is set to 5000 in "rsprompter_query_whu_config.py". The learning is also different (5e-4 v.s. 2e-4).

**Comment by KyanChen** [comment]
Thank you for your careful observation. In the experiment, we found that there is not much difference between 2e-4 and 5e-4. For anchor-based training, if the training time is too long, the effect will decrease. For query-based training, a longer training time will result in better performance.

**Comment by KyanChen** [comment]
We tried different training methods during the experiment, so there may be some differences.

**Comment by KyanChen** [comment]
Thank you, you can also have a try.
